### PR TITLE
Change category configuration

### PR DESCRIPTION
**Pull request by aledeg**
Before, we had a drop-down list to interract on categories. It was not
working the same way as feeds.
Now, categories and feeds behave in a similar manner. At the moment,
there is no change in features but it will allow to expand them.

See #2369

### CODE REVIEW

**Comment by aledeg** [comment]
Help on theme created with scss welcome!

File: app/i18n/en/sub.php [hunk 1]
```diff
@@ -14,7 +14,9 @@
 		'_' => 'Category',
 		'add' => 'Add a category',
 		'empty' => 'Empty category',
+		'informations' => 'Information',
```
Comment (Frenzie) [inline app/i18n/en/sub.php]: I believe it's all supposed to be English? ;-)

Comment (aledeg) [reply app/i18n/en/sub.php]: I know! I just copied it from the feed section.

Comment (aledeg) [reply app/i18n/en/sub.php]: I've changed everything to information!

**Comment by Alkarex** [comment]
@aledeg While waiting for an SCSS fan, I suggest you just write the same than your other CSS versions for now, both in the SCSS and in the CSS.

**Comment by aledeg** [comment]
@Alkarex After some headache, I figured out how to make sass work. :)

**Comment by Alkarex** [comment]
Very nice :-)
I have just made some minor whitespace changes.

**Comment by aledeg** [comment]
Thank you :blush:

**Pull request merged**